commands/add.js

```diff
@@ -4,25 +4,26 @@ const addTask = require('../database')
 const add = (newTask) => {
   fs.readFile('./tasks.json', 'utf-8', (error, data) => {
```

Comment:
consider moving the read/parse logic into a `loadTasks` function similar to your `addToFile` function that handles the writing to file.